sys/posix/pthread/include/pthread_spin.h

```diff
@@ -20,12 +20,17 @@
 #ifndef PTHREAD_SPIN_H
 #define PTHREAD_SPIN_H
 
-#include <stdatomic.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#ifdef __cplusplus
+#include <atomic>
+using namespace std;
```

Comment:
Not a good idea to do this in a header as the `using` statement leaks into the including file.

Reply:
It's clear to me and you are definitely right. It was just a workaround as described here: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=60932. I changed it already.